src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+import java.util.Scanner;
+import java.util.*;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
 public class Duke {
+//    private static final Task[] listTasks = new Task[100];
+    private static final ArrayList<Task> listTasks= new ArrayList<>(1);
+    public static int jobCount = 0;
+
+    /**
+     * Function to print Logo and Hello
+     */
+    static void printHello() {
+        String logo = "   _____  .__        \n"
+                + "  /  _  \\ |__|______ \n"
+                + " /  /_\\  \\|  \\_  __ \\\n"
+                + "/    |    \\  ||  | \\/\n"
+                + "\\____|__  /__||__|   \n"
+                + "        \\/           \n";
+        System.out.println("\tHello from\n" + logo + "\n");
+        System.out.println("____________________________________________________________\n");
+        System.out.println("\tSup! I'm Air\n" + "\tHow can I help you out today?\n");
+        System.out.println("____________________________________________________________\n");
+    }
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+    /**
+     * Function to print the list of tasks on typing list
+     */
+    static void printTaskList(int index) {
+        for (int i = 0; i < index; i++) {
+            System.out.println((i + 1) + ".  " + listTasks.get(i).printDetails());
+        }
+    }
+
+    /**
+     * Function to create task and add it to the listTasks
+     */
+
+    static void newTask(Task userTask) {
+        jobCount++;
+        listTasks.add(userTask);
+    }
+
+    private static void gibberishError() throws NoSuchCommandException {
```

Comment:
Method should sound like verbs. 

In any case, it seems odd that the function's only job is to throw the error. If you need to throw the error, why not just do it directly rather than calling this function?